include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -219,6 +212,11 @@ class MMR92C04 {
         can_client.send_can_message(get_host_id(), message);
     }
 
+    auto drdy_interrupt_response() -> void {
```

Comment:
I think this needs to be static here since we're calling it from the callback function. @sfoster1 correct me if I'm wrong.